package.json

```diff
@@ -13,6 +13,7 @@
     "babel-preset-metal": "^3.1.0",
     "babelify": "^7.2.0",
     "browserify": "^13.0.0",
+    "metal-component": "^1.0.0-rc.12",
```

Comment:
I was using 2.x, it complained about this missing dependency, so I added it. I've upgrade to npm 3. Should we leave it for the benefit of those on LTS releases?